### DISCUSSION TITLE
Force ride breakdown!

### DISCUSSION THREAD

**Discussion by tylere661** [OpenRCT2/OpenRCT2#19660] [Q&A] [answered]
The force ride breakdown feature is not found on my version in Openrct2.  I have version 0.4.3

**Comment by 733737** [comment]
it's there in the ride window

**Comment by ZeeMaji** [comment, marked as answer]
To be more specific, you need to have debug mode enabled in your settings and the option is under the ride window's maintenance tab. (a stop sign esc icon)
